add-tourpackage.php

```diff
@@ -178,158 +160,240 @@
+                                        </div>
+                                        <div class="col-12 mt-2">
+                                            <div class="row">
+                                                <div class="col-6 col-md-3 col-lg-3 mt-2">
+                                                    <center><img alt="" id="img-1" style="height: 150px; object-fit: cover;" class="img-fluid rounded-2 w-100" src="resources/images/default -image.svg"></center>
+                                                </div>
+                                                <div class="col-6 col-md-3 col-lg-3 mt-2">
+                                                    <center><img alt="" id="img-2" style="height: 150px; object-fit: cover;" class="img-fluid rounded-2 w-100" src="resources/images/default -image.svg"></center>
+                                                </div>
+                                                <div class="col-6 col-md-3 col-lg-3 mt-2">
+                                                    <center><img alt="" id="img-3" style="height: 150px; object-fit: cover;" class="img-fluid rounded-2 w-100" src="resources/images/default -image.svg"></center>
+                                                </div>
+                                                <div class="col-6 col-md-3 col-lg-3 mt-2">
+                                                    <center><img alt="" id="img-4" style="height: 150px; object-fit: cover;" class="img-fluid rounded-2 w-100" src="resources/images/default -image.svg"></center>
+                                                </div>
                                             </div>
                                         </div>
                                     </div>
                                 </div>
                             </div>
                         </div>
-                    </div>
-                    <div class="col-12 mt-3 mb-3">
-                        <div class="row">
-                            <div class="col-12 col-md-6 col-lg-6 mt-2">
-                                <button class="btn sbt-button">Clear</button>
+                        <div class="col-12 mt-3 mb-3">
+                            <div class="row">
+                                <div class="col-12 col-md-6 col-lg-6 mt-2">
+                                    <button class="btn sbt-button">Clear</button>
+                                </div>
+                                <div class="col-12 col-md-6 col-lg-6 mt-2">
+                                    <button class="btn sbt-button" onclick="addTourPackage();">ADD</button>
+                                </div>
                             </div>
-                            <div class="col-12 col-md-6 col-lg-6 mt-2">
-                                <button class="btn sbt-button" onclick="addTourPackage();">ADD</button>
+                        </div>
+
+                        <!-- destination selecttion model -->
```

Comment:
Do not create 3 models for each process. Create just one model and put all 3 design in it. Then you can manage this all 3 UIs using JavaScript.